mkt/users/models.py

```diff
@@ -147,7 +137,7 @@ def is_developer(self):
 
     @property
     def name(self):
-        return smart_unicode(self.display_name or self.username)
+        return smart_unicode(self.display_name or '-')
```

Comment:
I like `'user-%d' % self.id` actually, that's a good suggestion. I didn't want to have the name part of the email in there, because of the potential privacy concern. But exposing the user id is fine, I'll do that.